ga4gh/datamodel/datasets.py

```diff
@@ -82,6 +86,33 @@ def getVariantSetByIndex(self, index):
         """
         return self._variantSetIdMap[self._variantSetIds[index]]
 
+    def addPhenotypeAssociationSet(self, phenotypeAssociationSet):
+        """
+        Adds the specified g2p association set to this backend.
+        """
+        id_ = phenotypeAssociationSet.getId()
+        self._phenotypeAssociationSetIdMap[id_] = phenotypeAssociationSet
+        self._phenotypeAssociationSetNameMap[phenotypeAssociationSet.getLocalId()] = phenotypeAssociationSet
```

Comment:
Obviously these lines are too long. So the way this is done in most consistently in the rest of the code is
```python
self._phenotypeAssociationSetNameMap[
    phenotypeAssociationSet.getLocalId()] = phenotypeAssociationSet
```
We all like consistency...